app/models/concerns/pre_sale_prices/line_item_pre_saleable.rb

```diff
@@ -0,0 +1,20 @@
+module PreSalePrices
+  module LineItemPreSaleable
+    extend ActiveSupport::Concern
+
+    included do
+      before_validation :copy_pre_sale_price
```

Comment:
Shouldn't line items record pre sale price at time of creation? This would
allow better reporting.

On Thu, Jul 7, 2016 at 11:16 AM, Chris Blackburn <notifications@github.com>
wrote:

> In app/models/concerns/pre_sale_prices/line_item_pre_saleable.rb
> <https://github.com/madetech/spree_pre_sale_prices/pull/4#discussion_r69883173>
> :
>
> > @@ -0,0 +1,20 @@
> > +module PreSalePrices
> > +  module LineItemPreSaleable
> > +    extend ActiveSupport::Concern
> > +
> > +    included do
> > +      before_validation :copy_pre_sale_price
>
> Hmm, I was thinking this should always bring the pre-sale price in line
> with the current pre-sale price on the variant?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/madetech/spree_pre_sale_prices/pull/4/files/2bbe965b78f020763dce578b4fe22098246a6a40#r69883173>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAQpAIE3Yk5NOLf9HaCfcrgYIDDKYsLkks5qTNIDgaJpZM4JG6bg>
> .
>